R/select.R

```diff
@@ -0,0 +1,155 @@
+#' Select variables using `select`
```

Comment:
We try to document function names with `()`, so `select()` here
https://style.tidyverse.org/documentation.html#r-code

Reply:
I spotted a few things which weren't consistent across dyplr-based steps --> #660